modules/serverless-kn-func-emit.adoc

```diff
@@ -0,0 +1,70 @@
+[id="serverless-kn-func-emit_{context}"]
+= Emitting a test event to a deployed function
+
+You can use the `kn func emit` CLI command to emit a CloudEvent to a function that is either deployed locally or deployed to your {product-title} cluster. This command can be used to test that a function is working and able to receive events correctly.
+
+You can specify the following parameters for the emitted CloudEvent:
+
+Event type:: The type of event being emitted. You can find information about the `type` parameter that is set for events from a certain event producer in the documentation for that event producer. For example, the API server source event source may set the `type` parameter of produced events as `dev.knative.apiserver.resource.update`.
+Event source:: The event source that produced the event. This may be a URL for the event source, for example `https://10.96.0.1/`, or the name of the event source.
+Event ID:: A random unique ID that is created by the event producer.
+Event data:: The data contained in the event, if you are not sending events from an existing event source that already specifies data for the event. For example, if you are sending an event that uses the default settings, you can specify a `--data` value such as `"Hello world!"` so that the event will contain this data string. You can also specify what type of data is carried by the event, by using the `--content-type` flag.
```

Comment:
Should the `--content-type` flag be called out on its own? It seems odd that it's smashed in here with `--data`. Some of the language here is also a bit unclear to me. A clumsy suggestion, but hopefully useful.
```suggestion
Event data:: The event data. For example, you can specify a `--data` value such as `"Hello world!"` so that the event's data property is this string. By default, no data is included in the synthetic events created by `kn func emit`.
Data content type: You can also specify what type of data is carried by the event, by using the `--content-type` flag. In the example above, you might provide `text/plain`, e.g. `func emit --data "Hello world!" --content-type "text/plain"`.
```

Reply:
The reason they're together is because presumably you would only use the content type flag if you actually specify data as well, otherwise it wouldn't be relevant?

Reply:
Made some updates, lmk what you think